backend/src/main/backend/App.java

```diff
@@ -123,19 +123,37 @@ public static void main(String[] args) {
         */
         
         Database dataStore = Database.getDatabase(db_url);        
-        //PHASE 2
         
-        dataStore.dropTableUser();
-        dataStore.createTableUser();
+        Table userTable = new Table("tblUser");
+        Table messageTable = new Table("tblMessages");
+        Table commentTable = new Table("tblComment");
+        Table likesTable = new Table("tblLikes");
+
+
+        dataStore.dropTable(userTable);
+        dataStore.createTable(userTable); 
+
+        dataStore.dropTable(messageTable);
+        dataStore.createTable(messageTable); 
+        
+        dataStore.dropTable(commentTable);
+        dataStore.createTable(commentTable); 
+
+        dataStore.dropTable(likesTable);
+        dataStore.createTable(likesTable); 
+
+        
+        // dataStore.dropTableUser();
```

Comment:
I say this as someone that used to do this - don't comment out code, just delete it. Git will keep a record of it for you, and you'll find over time that you virtually never end up un-commenting out code.

Reply:
this is good to keep in mind. commented code has been deleted